components/script/dom/client.rs

```diff
@@ -47,6 +47,7 @@ impl Client {
         self.active_worker.get()
     }
 
+    #[allow(dead_code)]
```

Comment:
Note: this, or a variant of it, will probably be used as part of implementing https://github.com/servo/servo/issues/26378